centreon/www/front_src/src/CloudNotificationsConfiguration/EditPanel/PanelEdit.cypress.spec.tsx

```diff
@@ -524,3 +535,109 @@ describe('Edit Panel: Delete button', () => {
     cy.matchImageSnapshot();
   });
 });
+
+describe('Edit Panel: Duplicate button', () => {
+  beforeEach(initialize);
+
+  it('disables the Duplicate button when changes are made', () => {
+    cy.waitForRequest('@getNotificationRequest');
+
+    cy.findByTestId(labelDuplicate).should('not.be.disabled');
+
+    cy.findAllByTestId('CancelIcon').eq(0).click();
+
+    cy.findByTestId(labelDuplicate).should('be.disabled');
+
+    cy.matchImageSnapshot();
+  });
+
+  it('displays confirmation dialog with new notification name field on Duplicate button click', () => {
+    cy.waitForRequest('@getNotificationRequest');
+
+    cy.findByTestId(labelDuplicate).click();
+
+    cy.findByText(labelPleaseEnterNameForDuplicatedNotification).should(
+      'be.visible'
+    );
+    cy.findByLabelText(labelNotificationName).should('be.visible');
+    cy.findByText(labelDuplicate).should('be.disabled');
+    cy.findByText(labelDiscard).click();
+
+    cy.matchImageSnapshot();
+  });
+
+  it('validates that new name is required and not already taken', () => {
```

Comment:
validates that new name is present?

Reply:
what about : "validates that name field is required and not already taken" ?

Reply:
isn't it required in any case? or it really what the test does, check that it is required?

Reply:
yes, it checks if the field is required

Reply:
after discussion on Slack: not empty instead of required